pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -775,39 +775,37 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		switch port.Dhcp {
 		case types.DT_STATIC:
 			if port.AddrSubnet == "" {
-				errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s DT_STATIC but missing "+
-					"subnet address in %+v; ignored", port.IfName, port)
-				log.Error(errStr)
-				port.ParseError = errStr
-				port.ParseErrorTime = time.Now()
+				errStr := fmt.Sprintf("Port %s Configured as DT_STATIC but "+
+					"missing subnet address. Port: %+v.", port.IfName, port)
```

Comment:
%v for the whole port isn't useful to the admin. Maybe printing the sysAdapter.Addr in addition to the IfName would be helpful.

Reply:
Done.. Changed it to:

				errStr := fmt.Sprintf("Port %s Configured as DT_STATIC but "+
					"missing subnet address. SysAdapter - Name: %s, Addr:%s",
					port.IfName, sysAdapter.Name, sysAdapter.Addr)